### ISSUE TITLE
Entire block's content deleted on single backspace in Firefox when the plugin is active

### ISSUE THREAD

**Issue by cdevroe** [WordPress/gutenberg#13979]
**Describe the bug**
Clicking backspace to delete a single character results in an entire block's content being removed.

**To Reproduce**
1. Go to A New Post with the Gutenberg Editor
2. Begin typing anything into the empty block
3. Hit backspace
4. Entire block's content deleted.

**Expected behavior**
I would expect for a single character to be deleted.

**Desktop (please complete the following information):**
 - OS: Windows 10 Version 1809 Build 17763.316
 - Browser: Firefox
 - Version: 65.0

**Additional context**
- Gutenberg 5.1.0
- WordPress 5.0.3
- Firefox extensions: 1Password, LastPass, Inoreader

**Comment by earnjam** [comment]
After a git bisect, I determined that this was introduced in c17239fb5f5a09eae863880dc452a1b045539300

**Comment by designsimply** [comment]
Tested and confirmed using WordPress 5.0.3 and Gutenberg 5.1.0 hosted at https://jurassic.ninja/ using Firefox 65.0 on macOS 10.13.6.

Note: this problem only happens when the plugin is active and only in Firefox.

![13979-5s](https://user-images.githubusercontent.com/1119271/53112490-8f2ced80-34fc-11e9-89a5-25c35078d2be.gif)

Console log is getting the following error, looks like for every key press:

```
TypeError: t is undefined index.js:50:162547
```

cc @hypest @iseulde

**Comment by designsimply** [comment]
@earnjam c17239f should only affect the native mobile apps if I understand correctly.

**Comment by earnjam** [comment]
> @earnjam [c17239f](https://github.com/WordPress/gutenberg/commit/c17239fb5f5a09eae863880dc452a1b045539300) should only affect the native mobile apps if I understand correctly.

Yep, sorry updated my comment above. I discovered some specific circumstances of the issue that made it appear the commit before that was working.

**Comment by aduth** [comment]
Error occurs at this line:

https://github.com/WordPress/gutenberg/blob/f81d0d7dc41779a95d164098eabb219a95df5970/packages/editor/src/components/rich-text/index.js#L376

>TypeError: inputType is undefined[Learn More]

Quite likely related to changes introduced as part of #13833 (cc @iseulde @noisysocks )

**Comment by aduth** [comment]
Per the following, only Chrome has support for `InputEvent#inputType`:

https://developer.mozilla.org/en-US/docs/Web/API/InputEvent/inputType#Browser_compatibility

**Comment by aduth** [comment]
A not-so-great "workaround" (more to the point of confirming the above) is to enter this in [your browser console](https://developer.mozilla.org/en-US/docs/Tools/Web_Console/Opening_the_Web_Console) when loading the editor:

```js
window.addEventListener( 'input', ( e ) => e.inputType = '', true )
```

**Comment by aduth** [comment]
Fix proposed at #13986
